forge/lang/expander.rkt

```diff
@@ -803,14 +803,14 @@
 
 (define-syntax (ExampleDecl stx)
   (syntax-parse stx
-  [((~literal ExampleDecl) (~optional name:NameClass)
-                           pred:ExprClass
-                           bounds:BoundsClass)
-   (quasisyntax/loc stx
-     (example (~? name.name unnamed-example)
-       pred
+    [((~literal ExampleDecl) (~optional name:NameClass)
+                             pred:ExprClass
+                             bounds:BoundsClass)
+     (quasisyntax/loc stx
        (syntax-parameterize ([current-forge-context 'example])
-         (begin #,@(syntax/loc stx bounds.translate)))))]))
+         (example (~? name.name #,(make-temporary-name stx))
```

Comment:
No good, this breaks `add-to-execs`.

The example has to show up in a module context (according to `syntax-local-context`). Adding the parameterize changes it to a define context, which means lifting to the `execs` submodule won't work, which means the lifting doesn't happen, which means the example stays where it is and ends up getting provided.